toolkit/scalebar/src/main/java/com/arcgismaps/toolkit/scalebar/internal/ScalebarUtils.kt

```diff
@@ -118,15 +117,15 @@ internal object ScalebarUtils {
 
     /**
      * Formats a double to a string.
-     * If the double has no decimal part, it will be displayed as an int.
+     * If the double has no decimal part or the first two decimal places are 0, then it will be displayed as an int.
      * If the second digit after the decimal is zero, it will be displayed with one decimal place.
      * Otherwise, it will be displayed with two decimal places.
      *
      * @since 200.7.0
      */
     fun Double.format(): String {
         return when {
-            (this - this.toInt()).absoluteValue < 1e-12 -> this.toInt().toString() // Display as int if very close to an integer
+            this % 1.0 == 0.0 || (this * 100).toInt() % 100 == 0 -> this.toInt().toString()
             (this * 10) % 1.0 == 0.0 -> "%.1f".format(this) // Display one decimal place if last digit is zero
```

Comment:
This case will fail with the following: `10.101`
by casing `this` to `Int`, we can discard the rest of the degits after the first decimal place. This allows to check if the 1st decimal place is a 0 more precisely. 
```suggestion
            (this * 100).toInt() % 10 == 0  -> "%.1f".format(this)
```